lib/shoryuken/environment_loader.rb

```diff
@@ -15,9 +15,12 @@ def initialize(options)
     end
 
     def load
-      initialize_options
-      initialize_logger
-      load_rails if options[:rails]
+      if options[:rails]
+        load_rails
+      else
+        initialize_options
+        initialize_logger
```

Comment:
ERB can't be processed if using something from the Rails project, eg.
```
queues:
  - [<%= Rails.application.config.some_setting %>, 1]
  - [<%= Settings.some_other_setting %>, 1]
```
because options are initialised before the Rails environment is loaded. That config file results in an `uninitialized constant` error.